basics_dev/coding-style.md

```diff
@@ -165,7 +165,7 @@ Security coding guidelines
        height = untrusted_conf.height;
     ~~~
 
--   Use another variables, without the `untrusted_` prefix to hold the sanitized values, as shown above.
```

Comment:
What do you think is the best replacement for this line @jpouellet, if it is to stay in?